node_modules/gh-rest/lib/rest/user.js

```diff
@@ -98,7 +99,8 @@ module.exports = function(client) {
             'emailPreference': opts.emailPreference,
             'isAdmin': opts.isAdmin,
             'recaptchaChallenge': opts.recaptchaChallenge,
-            'recaptchaResponse': opts.recaptchaResponse
+            'recaptchaResponse': opts.recaptchaResponse,
+            'termsAndConditions': opts.termsAndConditions
```

Comment:
I guess this would also imply that we need to always check the terms and conditions on every request. I haven't seen the whole PR yet, but do we do a query to invalidate all user's terms and conditions when the T&C are updated?